gulpfile.js

```diff
@@ -170,7 +170,7 @@ gulp.task('stop-webserver', function() {
  * Adds a listener which will re-build on a file save.
  */
 gulp.task('default', function() {
-  runSequence('build', 'webserver', 'watch');
+  runSequence('offline', 'webserver', 'watch');
```

Comment:
Do we still need 2 separate tasks `build` and `offline`? The gulp file is getting big and messy and merging the offline task into the build one will make it a bit easier.